### ISSUE TITLE
JAR does not exist in BIntray or Manven central as described in the documentation

### ISSUE THREAD

**Issue by HunterSherms** [Kotlin/kotlinx.serialization#1143]
**Describe the bug**

Following [the documentation](https://github.com/Kotlin/kotlinx.serialization#maven) and including the dependency in maven like so:

```
<dependency>
    <groupId>org.jetbrains.kotlinx</groupId>
    <artifactId>kotlinx-serialization-json</artifactId>
    <version>1.0.0</version>
</dependency>
```

Maven is unable to find the dependency, with or without the bintray repository. After digging in a bit, visiting https://kotlin.bintray.com/kotlinx/org/jetbrains/kotlinx/kotlinx-serialization-json/1.0.0/ shows no JAR files available. There are, however, many sub modules that do appear to be available such as https://kotlin.bintray.com/kotlinx/org/jetbrains/kotlinx/kotlinx-serialization-json-jvm/1.0.0/

**To Reproduce**
Include `kotlinx-serialization-json` as a dependency in any project according to the readme documentation.

**Expected behavior**
MVN includes the dependency after following the documentation.

**Environment**
 - Kotlin version: latest
 - Library version: 1.0.0
 - Kotlin platforms: JVM
 - IDE version: IntelliJ latest

**Comment by h0tk3y** [comment]
It should work with Maven, too, as we publish a POM in the `kotlinx-serialization-json` module which has a single dependency on the actual module that contains the JVM artifact:
```
<dependencies>
  <dependency>
    <groupId>org.jetbrains.kotlinx</groupId>
    <artifactId>kotlinx-serialization-json-jvm</artifactId>
    <version>1.0.0</version>
    <scope>compile</scope>
  </dependency>
</dependencies>
```
Let me try to reproduce and investigate this...

**Comment by h0tk3y** [comment]
The default dependency declaration seems to be broken indeed when consumed from Maven (but not when consumed from Gradle versions that can't read Gradle module metadata).
As a temporary workaround, this version can be consumed with `<type>pom</type>` added to the dependency declaration.

**Comment by HunterSherms** [comment]
I was unable to get it working using `<type>pom</type>` - what eventually worked for me was using the `                <artifactId>kotlinx-serialization-json-jvm</artifactId>` directly

**Comment by sandwwraith** [comment]
@h0tk3y Is this a bug that can be fixed on our side or should we simply change maven instuctions?

**Comment by qwwdfsad** [comment]
It will be fixed in 1.4.20, no documentation updates required

**Comment by qwwdfsad** [comment]
Fixed in 1.1.0-RC
